install/03_rbac.yaml

```diff
@@ -25,11 +25,13 @@ rules:
   resources:
   - clusteroperators
   - clusteroperators/status
+  - infrastructures
   verbs:
   - create
   - get
   - update
   - list
+  - watch
```

Comment:
I would create a new entry for `infrastructures` where we only give `get/list/watch` (provided they are what we need). Granting write operations to the `infrastructures` objects is extra powers that we don't need in this case.

Reply:
Makes sense. Updated.